core/core/test/serverspec/spec/kafka/kafka_spec.rb

```diff
@@ -1,28 +1,120 @@
 require 'spec_helper'
 
-
-describe service('kafka') do
-  it { should be_enabled }
-  it { should be_running }
+describe 'Checking if kafka service is up' do
+  describe service('kafka') do
+    it { should be_enabled }
+    it { should be_running }
+  end
 end
 
-describe port(9092) do
-  it { should be_listening }
+describe 'Checking user configuration' do
+  describe group('kafka') do
+    it { should exist }
+  end
+  describe user('kafka') do
+    it { should exist }
+    it { should belong_to_group 'kafka' }
+    it { should have_home_directory '/home/kafka' }
+    it { should have_login_shell '/sbin/nologin' }
```

Comment:
Should be: /usr/sbin/nologin
It'll work on RHEL7 because its /sbin is linked to /usr/sbin. On Ubuntu it's not linked (at least Azure images of 18.04).

Either way /usr/sbin should be safe for distros made in the last 5 years or so.

Reply:
Please note this change: https://github.com/epiphany-platform/epiphany/pull/97